app/move/controllers/create/save.js

```diff
@@ -110,7 +110,8 @@ class SaveController extends CreateBaseController {
   }
```

Comment:
See codeclimate is complaining about missing coverage…

Would it make sense to have this as the last commit after the other changes?

Reply:
Yep, this was just a draft, no test coverage updates yet.